### DISCUSSION TITLE
#36. VATZ biweekly meeting at `2023-06-21 22:00:00 +09:00:00`

### DISCUSSION THREAD

**Discussion by xellos00** [dsrvlabs/vatz#471] [General]
### 1. Overall

<img width="1330" alt="image" src="https://github.com/dsrvlabs/vatz/assets/6308023/c39a77fe-964d-49dc-acdb-cc1b9a3f7f28">

### 2. Statistic Rate
 
Sprint | Issue fulfillment | progress rate(%)
:--: | :--: | :--:
34 | 6/22 | 27
35 | 4/30 | 13

---

<img width="1021" alt="image" src="https://github.com/dsrvlabs/vatz/assets/6308023/b6bdf686-5af5-4cef-b27c-76d9553fdcc0">

**Comment by xellos00** [comment]
# 동요

- https://github.com/dsrvlabs/vatz/issues/451
    - Documentation
       - https://github.com/dsrvlabs/vatz/issues/470
       - https://github.com/dsrvlabs/vatz/issues/462
       - https://github.com/dsrvlabs/vatz/issues/467
       - https://github.com/dsrvlabs/vatz/issues/464
    - Bug fix 
       - https://github.com/dsrvlabs/vatz/issues/469
       - https://github.com/dsrvlabs/vatz/issues/466
       - https://github.com/dsrvlabs/vatz/issues/463

**Comment by rootwarp** [comment]
# 준교
N/A

**Comment by gnongs** [comment]
## 진홍
- No issue

**Comment by skonhwang** [comment]
# 승곤
- no issue

**Comment by qkrybin** [comment]
## 영빈 
- #454

**Comment by kim201212** [comment]
# 형기

- no issue

**Comment by xellos00** [comment]
<img width="339" alt="image" src="https://github.com/dsrvlabs/vatz/assets/6308023/553c21a4-9c34-4d2b-85a9-32f74cc1012c">

**Reply by xellos00** [reply]
<img width="360" alt="image" src="https://github.com/dsrvlabs/vatz/assets/6308023/b39684f8-37a0-46c3-8b76-89e642162ba6">

**Reply by xellos00** [reply]
@qkrybin 
Please, tweet our VATZ release on (KST) 2023-6-23 22:00:00

<img width="338" alt="image" src="https://github.com/dsrvlabs/vatz/assets/6308023/562a5cbc-6df9-4eb8-bdbb-5508db1e8aac">
